src/data.js

```diff
@@ -5639,6 +5639,27 @@ module.exports = [
       'Python',
     ],
   },
+  {
+    name: 'Andrei Racasan',
+    description: 'Full Stack Developer with a passion for finding pragmatic solutions to technical challenges.',
+    url: 'https://www.andreiracasan.com/about',
```

Comment:
Hey @0xAMR, will you please add a page with the requirements for the URL?

`https://www.andreiracasan.com/about`

`use|uses|using|setup|environment` at the end instead of about.

Reply:
My apologies, please see now.